packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -82,6 +90,50 @@ class ExpansionPanel {
   /// The widget builder that builds the expansion panels' header.
   final ExpansionPanelHeaderBuilder headerBuilder;
 
+  /// The widget builder that builds a custom expansion icon to expand
```

Comment:
A callback that returns a widget which shows if the panel is expanded or not.

Reply:
Done